handlers.go

```diff
@@ -3600,6 +3604,67 @@ func (s *Server) handleDeleteCollection(c echo.Context, u *User) error {
 	return c.NoContent(http.StatusOK)
 }
 
+// handleDeleteContentFromCollection godoc
+// @Summary      Deletes a content from a collection
+// @Description  This endpoint is used to delete an existing content from an existing collection. If two or more files with the same contentid exist in the collection, delete the one in the specified path
+// @Tags         collections
+// @Param        coluuid path string true "Collection ID"
+// @Param        contentid path string true "Content ID"
+// @Param        path query string false "Path in which delete files (if found)"
+// @Produce      json
+// @Success      200  {object}  string
+// @Failure      400  {object}  util.HttpError
+// @Router       /collections/{coluuid}/content/{contentid} [delete]
+func (s *Server) handleDeleteContentFromCollection(c echo.Context, u *util.User) error {
+	coluuid := c.Param("coluuid")
+	contentid := c.Param("contentid")
+	path := c.QueryParam("path")
+
+	col, err := collections.GetCollection(coluuid, s.DB, u)
+	if err != nil {
+		return err
+	}
+
+	content, err := util.GetContent(contentid, s.DB, u)
+	if err != nil {
+		return err
+	}
+
+	refs := []collections.CollectionRef{}
+	if err := s.DB.Model(collections.CollectionRef{}).
+		Where("collection = ?", col.ID).
+		Where("content = ?", content.ID).
+		Scan(&refs).Error; err != nil {
+		return err
+	}
+
+	if len(refs) > 1 {
+		// file is present more than once in collection, need path to figure out which exactly to delete
+		if path == "" {
```

Comment:
this means the path is a required field, if one wants to remove a specific content (unique estuary content ID), they need not send a path.